tests/Action/GetShortObjectDescriptionActionTest.php

```diff
@@ -150,14 +139,10 @@ public function testGetShortObjectDescriptionActionObject(): void
         $this->assertSame('renderedTemplate', $response->getContent());
     }
 
-    /**
-     * NEXT_MAJOR: Expect a NotFoundHttpException instead.
-     *
-     * @group legacy
-     * @expectedDeprecation Trying to get a short object description for a non found object is deprecated since sonata-project/admin-bundle 3.x and will be throw a 404 in version 4.0.
-     */
     public function testGetShortObjectDescriptionActionEmptyObjectIdAsJson(): void
     {
+        $this->expectException(NotFoundHttpException::class);
```

Comment:
Can you move this just before where the exception will be triggered?

Reply:
In other tests I looked, this is often the first line of the test.
I don't see this like a bad thing, you see easier what's expected in this test.

Reply:
On several PRs I have been asked to change the same thing. It makes sense to move it right before because if you dont do it and other things trigger an exception things might go wrong.